schema/handler.go

```diff
@@ -77,34 +80,140 @@ type getJsonSchema struct {
 //       200: jsonSchema
 //       404: jsonError
 //       500: jsonError
-func (h *Handler) get(w http.ResponseWriter, r *http.Request, ps httprouter.Params) {
+func (h *Handler) getByID(w http.ResponseWriter, r *http.Request, ps httprouter.Params) {
 	s, err := h.r.IdentityTraitsSchemas(r.Context()).GetByID(ps.ByName("id"))
 	if err != nil {
 		h.r.Writer().WriteError(w, r, errors.WithStack(herodot.ErrNotFound.WithDebugf("%+v", err)))
 		return
 	}
-	var src io.ReadCloser
 
-	if s.URL.Scheme == "file" {
-		src, err = os.Open(s.URL.Host + s.URL.Path)
+	ls, err := readSchema(s)
+	if err != nil {
+		h.r.Writer().WriteError(w, r, errors.WithStack(herodot.ErrInternalServerError.WithReasonf("The file for this JSON Schema ID could not be found or opened. This is a configuration issue.").WithDebugf("%+v", err)))
+		return
+	}
+
+	w.Header().Add("Content-Type", "application/json")
+	enc := json.NewEncoder(w)
+	enc.SetIndent("", "  ")
+	if err := enc.Encode(ls); err != nil {
+		h.r.Writer().WriteError(w, r, errors.WithStack(herodot.ErrInternalServerError.WithReasonf("The file for this JSON Schema ID could not be found or opened. This is a configuration issue.").WithDebugf("%+v", err)))
+		return
+	}
+}
+
+// Raw JSON Schema map
+//
+// swagger:model jsonSchemas
+// nolint:deadcode,unused
+type jsonSchemas map[string]json.RawMessage
```

Comment:
It makes sense from a data structure perspective to return a map of schema ID to schema, but that will mean that requesting the same page will include the same schemas but in a different order. I know that JSON objects have no order, but I would expect a list API to return something ordered. Maybe make this an array instead? Not sure though, just a thought.

Reply:
I see your point, but i chose a map because the schema json doesn't include the id of the schema itself. I don't know if there is a solution to this problem

Reply:
You could also return
```json
[{"id": "default", "schema": {...}}, {"id": "other", "schema": {...}}]
```

Reply:
Yeah I'll do it that way